### PR TITLE
New Recipe: AM v13.0.0

### PR DESCRIPTION
**Pull request by iniyannatarajan**

### CODE REVIEW

File: A/AM/build_tarballs.jl [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+    ArchiveSource("https://zenodo.org/records/8161261/files/am-13.0.tgz", "1298257dc6a1f50aabc495bc7055e27975f03b67bb28e8f59f812ff39ddd3a36")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/am-13.0/src/
+make am
+install -Dvm 755 "am${exeext}" "${bindir}/am${exeext}"
+install_license ./LICENSE
+exit
```
Comment (giordano) [inline A/AM/build_tarballs.jl:19]:
```suggestion
```

File: A/AM/build_tarballs.jl [hunk 2]
```diff
@@ -0,0 +1,52 @@
+# Note that this script can accept some limited command-line arguments, run
+# `julia build_tarballs.jl --help` to see a usage message.
+using BinaryBuilder, Pkg
+
+name = "AM"
```
Comment (giordano) [inline A/AM/build_tarballs.jl:5]: I take this package doesn't have a more descriptive name, does it?

File: A/AM/build_tarballs.jl [hunk 3]
```diff
@@ -0,0 +1,52 @@
...
+
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://zenodo.org/records/8161261/files/am-13.0.tgz", "1298257dc6a1f50aabc495bc7055e27975f03b67bb28e8f59f812ff39ddd3a36")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/am-13.0/src/
+make am
```
Comment (giordano) [inline A/AM/build_tarballs.jl:16]: Can we build in parallel, to speed up the job?
```suggestion
make -j ${nproc} am COMPILER_GCC=cc
```

File: A/AM/build_tarballs.jl [hunk 4]
```diff
@@ -0,0 +1,52 @@
...
+    Platform("powerpc64le", "linux"; libc = "glibc"),
+    Platform("i686", "linux"; libc = "musl"),
+    Platform("x86_64", "linux"; libc = "musl"),
+    Platform("aarch64", "linux"; libc = "musl"),
+    Platform("armv6l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("armv7l", "linux"; call_abi = "eabihf", libc = "musl"),
+    Platform("x86_64", "macos"; ),
+    Platform("aarch64", "macos"; )
+]
+
```
Comment (giordano) [inline A/AM/build_tarballs.jl:39]: What's the problem with the other platforms?
```suggestion
platforms = supported_platforms()
```

File: A/AM/build_tarballs.jl [hunk 5]
```diff
@@ -0,0 +1,52 @@
...
+
+# The products that we will ensure are always built
+products = [
+    ExecutableProduct("am", :am)
+]
+
+# Dependencies that must be installed before this package can be built
+dependencies = Dependency[
```
Comment (giordano) [inline A/AM/build_tarballs.jl:47]:
```suggestion
dependencies = [
```

File: A/AM/build_tarballs.jl (hunk 2 above)
Comment (iniyannatarajan) [reply A/AM/build_tarballs.jl:5]: No, it does not have a more descriptive name.

File: A/AM/build_tarballs.jl (hunk 3 above)
Comment (iniyannatarajan) [reply A/AM/build_tarballs.jl:16]: With the "COMPILER_GCC=cc" flag, the build fails on FreeBSD and Apple systems. The AM codebase uses only gcc for compilation and the default Clang compiler in FreeBSD and Apple seem to fail. The parallel build succeeds on all systems with this flag removed.

**Comment by iniyannatarajan** [comment]
@giordano The windows builds seem to pass the building stage but fail in the audit stage with the message:
```
[ Info: Beginning audit of /cache/build/yggy-amdci7-2/julialang/yggdrasil/A/AM/build/i686-w64-mingw32/9d7lCI3L/i686-w64-mingw32-libgfortran3-cxx03/destdir
[ Info: Checking bin/am with RPath list String[]
[ Info: Ignored system libraries msvcrt.dll, KERNEL32.dll, libgomp-1.dll
[ Info: Checking license file
[ Info: Found license file(s): LICENSE
[ Info: ["am"] does not exist, reporting unsatisfied
┌ Error: Built AM but am still unsatisfied:
└ @ BinaryBuilder /cache/julia-buildkite-plugin/depots/e2fd9734-29d8-45cd-b0eb-59f7104f3131/packages/BinaryBuilder/L8tBh/src/AutoBuild.jl:946
ERROR: LoadError: Cannot continue with unsatisfied build products!
```
This does not happen when I run the Binary Builder wizard on my Linux machine. Do you have any suggestions for avoiding this error? Thanks!

File: A/AM/build_tarballs.jl [hunk 6]
```diff
@@ -0,0 +1,36 @@
...
+# Collection of sources required to complete build
+sources = [
+    ArchiveSource("https://zenodo.org/records/8161261/files/am-13.0.tgz", "1298257dc6a1f50aabc495bc7055e27975f03b67bb28e8f59f812ff39ddd3a36")
+]
+
+# Bash recipe for building across all platforms
+script = raw"""
+cd $WORKSPACE/srcdir/am-13.0/src/
+make -j ${nproc} am
+install -Dvm 755 "am" "${bindir}/am"
```
Comment (giordano) [inline A/AM/build_tarballs.jl:17]:
```suggestion
install -Dvm 755 "am" "${bindir}/am${exeext}"
```

**Pull request merged**